### ISSUE TITLE
Feature Request: 'Add New' button at the bottom of UI for mass calculations.

### ISSUE THREAD

**Issue by MongolianMisfit** [wpninjas/ninja-forms#3074]
As the title suggests, the user requests: 

_~"When you're adding many
calculations it gets difficult to add more calculations, since each time you
need to scroll to top, click Add New, and it automatically scrolls to
bottom. You should include a Add New button on the bottom also."_

Link to original ticket: https://wpninjas.kustomerapp.com/app/customers/59a680acb848a10010fa6316/event/59bf9afc7518100011f7449c

**Comment by ivankozlik** [comment]
Yes! The same thing is with Conditional Logic! (but it does not scroll automatically so that actually helps get more fields quickly)
